rosdep/python.yaml

```diff
@@ -6874,6 +6874,11 @@ python3-wxgtk4.0:
   fedora: [python3-wxpython4]
   gentoo: [dev-python/wxpython]
   ubuntu: [python3-wxgtk4.0]
+python3-xmltodict:
+  alpine: [py3-xmltodict]
+  arch: [python-xmltodict]
+  debian: [python3-xmltodict]
+  ubuntu: [python3-xmltodict]
```

Comment:
Can you also add a fedora entry, like:

```
fedora: [python3-xmltodict]
```

Reply:
https://fedora.pkgs.org/32/fedora-x86_64/python3-xmltodict-0.12.0-7.fc32.noarch.rpm.html

Reply:
I added fedra package in https://github.com/ros/rosdistro/pull/27499/commits/404adf31b1185b08a0f6098d5aef3979c174f99f